requirements.txt

```diff
@@ -18,7 +18,7 @@ rdp==0.8
 requests==2.24.0
 retry==0.9.2
 shapely==1.7
-urllib3==1.25.9
+urllib3==1.26.5
```

Comment:
There is no download/urllib used in this PR. Can this change be removed or moved to a PR that needs this?

Reply:
This change came to effect when I pulled the upstream master into this branch.